pkg/osquery/tables/wifi_networks/wifi_networks.go

```diff
@@ -0,0 +1,149 @@
+package wifi_networks
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/launcher/pkg/osquery/tables/dataflattentable"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/kolide/launcher/pkg/osquery/tables/wifi_networks/internal"
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+//go:generate go-bindata -nometadata -nocompress -pkg internal -o internal/assets.go internal/assets/
+
+type execer func(ctx context.Context, buf *bytes.Buffer) error
+
+type WlanTable struct {
+	client    *osquery.ExtensionManagerClient
+	logger    log.Logger
+	tableName string
+	getBytes  execer
+}
+
+func TablePlugin(client *osquery.ExtensionManagerClient, logger log.Logger) *table.Plugin {
+	columns := dataflattentable.Columns(
+		table.TextColumn("ssid"),
+	)
+
+	t := &WlanTable{
+		client:    client,
+		logger:    logger,
+		tableName: "kolide_wifi_networks",
+		getBytes:  execPwsh(logger),
+	}
+
+	return table.NewPlugin(t.tableName, columns, t.generate)
+}
+
+func (t *WlanTable) generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+	var results []map[string]string
+	var output bytes.Buffer
+
+	err := t.getBytes(ctx, &output)
+	if err != nil {
+		return results, err
+	}
+	scanner := bufio.NewScanner(&output)
+	scanner.Split(tablehelpers.StanzaSplitter)
+	for scanner.Scan() {
+		chunk := scanner.Bytes()
+		rows, err := dataflatten.Ini(chunk, dataflatten.WithLogger(t.logger))
+		if err != nil {
+			return results, errors.Wrap(err, "flattening data")
+		}
+		ssid := ""
+		for _, r := range rows {
+			if strings.HasSuffix(r.StringPath("/"), "/SSID") {
+				ssid = r.Value
+				break
+			}
+		}
+		rowData := map[string]string{
+			"ssid": ssid,
+		}
```

Comment:
What's the `ssid` pull out for? It's in the EAV data, extracting it here feels superfluous?

Reply:
Hrm. This data looks suspect. What delineates the networks?

Reply:
Ah. I'm starting to see what's happening -- these are objects, which powershell spits out in something close to ini, but each object is empty line delineated. And that means ini parses them as a big honking object which keeps overwriting itself. 

I don't know if there's a simple ini or powershell patch, but it would probably be simple to add section headers in go. Using an incrementing integer would work.

Reply:
Found it! `ConvertTo-Json`. See the powershell script

Reply:
> And that means ini parses them as a big honking object which keeps overwriting itself.

I don't think that's the case, because this code passes each empty-line-delineated chunk to the ini parser separately.